src/runtime/tests/manifest-parser-test.ts

```diff
@@ -661,4 +662,292 @@ describe('manifest parser', () => {
       '[{"type":"artist","name":"in this moment"}]'
     );
   });
+
+  describe('inline data stores', () => {
+    // Strips 'location' fields from the AST.
+    function clean(manifestAst) {
+      return JSON.parse(JSON.stringify(manifestAst, (k, v) => k === 'location' ? undefined : v));
+    }
+
+    it('parses text fields', () => {
+      const manifestAst = clean(parse(`
+        store A of {txt: Text} with {
```

Comment:
Is there a way to distinguish between a collection and a singleton store?

Shouldn't store `of {txt: Text}` or `of [{txt: Text}]` ? Maybe not for this CL, but I am wondering whether you can declare singleton stores with this syntax

Reply:
This is something I need to chase up. It doesn't seem to matter which type you use for resource-based stores, and that's confusing.

Reply:
I'll also update the tests to use collection types where appropriate, pending further investigation.